docs/functions/functions.json

```diff
@@ -437,6 +437,77 @@
 				true
 			]
 		},
+		{
+			"name": "max",
+			"id": "max",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Calculate the maximum of all numbers in the list.",
+			"argument": [
+				{
+					"name": "list",
+					"optional": false,
+					"type": "List of numbers",
+					"description": "The numbers you are trying to find the max of."
+				}
```

Comment:
Actually, it also accept multiple params, like `max(1,2,3)`. We should mention that somewhere.

Reply:
I will mention that for `min` as well